.github/pull_request_template.md

```diff
@@ -1,67 +1,38 @@
-NOTICE: Delete the sections that do not apply to your PR, and fill out the section that does.
-(You can open a PR to add or improve a section, if you find them lacking!)
+(The lines in brackets (like this one) is instructions for filling out this template. These lines can be deleted.)
+(The lines in double curly brackets ({{}}) should be replaced as it describes.)
+(You can open a PR to add or improve this template, if you find it lacking!)
 
-----------------------
+(Choose one heading, or add your own.)
 **Content (Artwork / Missions / Jobs)**
+**Balance**
+**Feature**
+**Bug fix**
+**CI/CD/Testing**
+**Documentation**
 
-## Summary
-{{summarize your content! Include links to related issues, in-game screenshots, etc.}}
-
-## Save File
-This save file can be used to play through the new mission content:
-{{attach a save file that allows people to easily test your added mission content or see your new in-game art}}
-
-## Artwork Checklist
- - [ ] I updated the copyright attributions, or decline to claim copyright of any assets produced or modified
- - [ ] I created a PR to the [endless-sky-assets repo](https://github.com/endless-sky/endless-sky-assets) with the necessary image, blend, and texture assets: {{insert PR link}}
- - [ ] I created a PR to the [endless-sky-high-dpi repo](https://github.com/endless-sky/endless-sky-high-dpi) with the `@2x` versions of these art assets: {{insert PR link}}
+This PR addresses the bug/feature described in issue #{{insert number}}
 
+## Summary
+{{Describe and explain your changes. Include links to related issues.}}
 
------------------------
-**Bugfix:** This PR addresses issue #{{insert number}}
+## Screenshots
+{{Include before and after screenshots demonstrating your changes, if applicable.}}
 
-## Fix Details
-{{add details}}
+## Usage examples
+{{If your changes affect how game data can be defined, provide examples demonstrating the changes here.}}
 
 ## Testing Done
-{{describe how you tested that the fix doesn't introduce other issues}}
+{{Describe how you tested these changes. How they meet the purpose. Ensure they don't introduce new issues.}}
```

Comment:
What does "how they meet the purpose" mean? Is it in relation to the issue this PR addresses or whether the test itself works?

Reply:
```suggestion
{{Describe how you tested these changes. Ensure that new issues aren't introduced.}}
```